NuRadioReco/modules/channelCWNotchFilter.py

```diff
@@ -0,0 +1,227 @@
+import numpy as np
+from scipy import signal
+from NuRadioReco.utilities import units
+from NuRadioReco.utilities import fft
+
+"""
+Contains function to filter continuous wave out of the signal
+functions should work on a per event basis to comply with the iteration methods used in readRNOGData
+"""
+
+
+def find_frequency_peaks_from_trace(trace : np.ndarray, fs : float = 3.2e9 * units.Hz, threshold = 4):
+    """
+    Function fo find the frequency peaks in the real fourier transform of the input trace,
+
+    Parameters
+    ----------
+    trace : np.ndarray
+        waveform (shape: [24,2048])
+    fs: float, default = 3.2e9 Hz
+        sampling frequency of the RNO-G DAQ, (input should be taking from the channel object)
```

Comment:
In line with the comment above I would not refer here to RNO-G. I would probably also remove the default value.